schemas/config-editor.xsd

```diff
@@ -73,7 +77,6 @@ a set of views with at least one defined view.
         <xs:element minOccurs="0" maxOccurs="1" ref="multilingualFields"/>
         <xs:element minOccurs="0" maxOccurs="1" ref="tableFields"/>
         <xs:element minOccurs="1" maxOccurs="1" ref="views"/>
```

Comment:
`snippets` was added in https://github.com/geonetwork/core-geonetwork/pull/7263 - we need to keep it:

```suggestion
        <xs:element minOccurs="1" maxOccurs="1" ref="views"/>
        <xs:element minOccurs="0" maxOccurs="1" ref="snippets"/>
```